cves/CVE-2015-1831.yml

```diff
@@ -173,11 +176,11 @@ lessons:
     applies:
     note:
   native_wrappers:
-    applies:
-    note:
+    applies: true
+    note: It was because of a function that overrode the DefaultExcludedPatternsChecker that made the the hole appear.
```

Comment:
Consider using a term other than "hole" to define the vulnerability

Reply:
Description has been updated